Meta/lint-ports.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import os
+import re
+
+
+PORT_TABLE_REGEX = re.compile('^\\| \\[`([^`]+)`\\][^`]+$')
```

Comment:
nits:

1. Use a raw string (`r'^\| \[`([^`]+)`\][^`]+$'`)
2. Add a comment like `# Matches e.g. "| [``bash``]..." and captures "bash" in group 1